packages/common/src/ssh/base-client.ts

```diff
@@ -44,8 +44,13 @@ export const parseSshUrl = (s: string): Pick<SshConnectionConfig, 'hostname' | '
 export type HelloResponse = {
   clientId: string
   tunnels: Record<string, string>
-  // TODO: baseUrl should be a string in the next deployment of the tunnel service, this is for backwards compat
-  baseUrl: string | { hostname: string; port: string; protocol: string }
+  rootUrl: string
+}
+
+// TODO: temporary, baseUrl should be a replaced with rootUrl after the next deployment of the tunnel service
+type LegacyBaseUrl = { hostname: string; port: string; protocol: string }
```

Comment:
There's no need to merge code with branching on baseUrl/RootUrl as the server should support both cases after deployment (and before, it just support v0.0.35)